codestyle/druid-forbidden-apis.txt

```diff
@@ -1,3 +1,4 @@
 com.google.common.collect.MapMaker @ Create java.util.concurrent.ConcurrentHashMap directly
 com.google.common.collect.Maps#newConcurrentMap() @ Create java.util.concurrent.ConcurrentHashMap directly
-com.google.common.util.concurrent.Futures.transform(com.google.common.util.concurrent.ListenableFuture<I>, com.google.common.util.concurrent.AsyncFunction<? super I,? extends O>) @ Use io.druid.java.util.common.concurrent.ListenableFutures.transformAsync
+com.google.common.util.concurrent.Futures.transform(com.google.common.util.concurrent.ListenableFuture<I>, com.google.common.util.concurrent.AsyncFunction<? super I,? extends O>) @ Use io.druid.java.util.common.concurrent.ListenableFutures#transformAsync
```

Comment:
This line uses different method referencing syntax than the other lines (`Futures.transform` vs. `Maps#newConcurrentMap`). Did you check that this prohibition works by trying to add a use of prohibited method to code and then `mvn install` to see if the build fails?

Reply:
fixed